osu.Game/Overlays/Comments/DrawableComment.cs

```diff
@@ -449,5 +438,31 @@ private string getParentMessage()
                 return parentComment.HasMessage ? parentComment.Message : parentComment.IsDeleted ? @"deleted" : string.Empty;
             }
         }
+
+        private class Loading : Container
```

Comment:
does this really need to be a dedicated class? couldn't you just have the `Container` in the hierarchy in `load()` and just operate on the container itself instead of adding all this?

Reply:
Loading is always hidden on start and I didn't wan to add more stuff to `DrawableComment`

Reply:
I still don't think splitting this out is worth it instead of just inlining the `LoadingSpinner` inside of `DrawableComment` as this is just one tree level extra. the new fillflow added also in this PR is more of a concern to me due to the depth of the hierarchy in that spot.